Tasks/IISWebAppDeployment/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
-        "Patch": 5
+        "Minor": 1,
```

Comment:
This I have added to be able upload in my local tfs.. I will change it to 0 before checkin